docs/usage/help.rst.inc

```diff
@@ -167,6 +167,16 @@ Examples::
     a directory, it won't recurse into it and won't discover any potential matches for
     include rules below that directory.
 
+    .. note::
+
+        It's possible that a sub-directory/file is matched while parent directories are not.
+        In that case, parent directories are not backed up thus their user, group, permission,=
+        etc. can not be restored.
+
+        For example, with `--pattern=+/home/user/subdir --pattern=-/home/user/*`,
+        `/home/user/subdir` can be restored with correct metadata, while `\home`and
+        `/home/user` can not.
+
```

Comment:
guess that is a rather special scenario.

the more general scenario is that a deep fs object is extracted while the parent dir(s) are not extracted (either because they were not chosen to get extracted or they are not even present in the archive because they were not backed up).

and borg just can't set metadata for stuff it does not extract from an archive entry, because it does not know that metadata in that case.

Reply:
While googling about this directory metadata not get restored problem, I saw issue #1751. I'm perfectly fine with current behavior, just guess it's better to mention this in the doc.

Maybe mention this also in `borg extract` doc?

Reply:
yeah, i agree it should be in the docs. borg extract looks like a good place.

Reply:
Added some comments for the behavior in `borg extract` doc. I'm not native English speaker, please feel free to change the description if it's not clear.